chapters/ch42.md

```diff
@@ -1,35 +1,37 @@
 > # 42. Addressing data mismatch
 
---> _replace THIS LINE by your translation for the above line_
+# 42. Xử lý dữ liệu không tương đồng
 
 > Suppose you have developed a speech recognition system that does very well on the training set and on the training dev set. However, it does poorly on your dev set: You have a data mismatch problem. What can you do?
 
---> _replace THIS LINE by your translation for the above line_
+Giả sử bạn đã phát triển một hệ thống nhận dạng giọng nói hoạt động rất tốt  trên tập huấn luyện và trên tập huấn luyện - phát triển. Tuy nhiên, hệ thống đó lại hoạt động kém trên tập phát triển: Bạn có vấn đề về dữ liệu không tương đồng. Bạn có thể làm gì?
 
 
 > I recommend that you: (i) Try to understand what properties of the data differ between the training and the dev set distributions. (ii) Try to find more training data that better matches the dev set examples that your algorithm has trouble with. [14]
 
---> _replace THIS LINE by your translation for the above line_
+Tôi đề xuất bạn nên: (i) Cố gắng hiểu những thuộc tính nào của dữ liệu là khác nhau giữa phân phối của tập huấn luyện và tập phát triển. (ii) Cố gắng tìm thêm dữ liệu huấn luyện tương đồng hơn với những mẫu trong tập phát triển mà thuật toán của bạn đang gặp vấn đề.
 
 
 > For example, suppose you carry out an error analysis on the speech recognition dev set: You manually go through 100 examples, and try to understand where the algorithm is making mistakes. You find that your system does poorly because most of the audio clips in the dev set are taken within a car, whereas most of the training examples were recorded against a quiet background. The engine and road noise dramatically worsen the performance of your speech system. In this case, you might try to acquire more training data comprising audio clips that were taken in a car. The purpose of the error analysis is to understand the significant differences between the training and the dev set, which is what leads to the data mismatch.
 
---> _replace THIS LINE by your translation for the above line_
+Ví dụ, giả sử bạn thực hiện phân tích lỗi nhận dạng giọng nói trên tập phát triển: Bạn duyệt qua 100 mẫu một cách thủ công và cố gắng hiểu xem thuật toán đang phạm lỗi ở những đâu. Bạn phát hiện rằng hệ thống của bạn thực thi kém vì hầu hết những đoạn âm thanh trong tập phát triển được thu trong ô tô, trong khi hầu hết các ví dụ để huấn luyện được thu trên một nền yên tĩnh. Tiếng ồn từ động cơ và đường phố làm ảnh hưởng nghiêm trọng đến chất lượng của hệ thống của bạn. Trong trường hợp này, bạn có thể cố gắng thu thập thêm dữ liệu huấn luyện bao gồm những đoạn âm thanh được thu trong ô tô. Mục đích của việc phân tích lỗi là để hiểu những khác biệt đáng kể giữa tập huấn luyện và tập phát triển, vốn là nguyên nhân dẫn đến dữ liệu không tương đồng.
```

Comment:
* Chỗ "phạm lỗi" nghe hơi nghiêm trọng, "có lỗi" thì thế nào?
* "thực thi" -> "hoạt động"?
* "car" được dịch là "xe hơi" ở vài chương trước, nên thống nhất chọn một cách. Mình đề xuất là "xe hơi"
* "nền yên tĩnh" -> "môi trường yên tĩnh"?

Reply:
👍 

- --> "Mắc lỗi"? (thuật toán "có lỗi" nghe giống như faulty, là một intrinsic characteristic vậy. "Mắc" nghe giống vấp phải lỗi, gây ra lỗi trong quá trình hoạt động hơn. Một thuật toán "có lỗi" trong bản chất vẫn có xác suất "may" khi thực hiện không mắc lỗi nào.)

- "Hoạt động" nghe tự nhiên hơn ổn hơn, hơn nữa "thực thi" thường phải có object chứ không đi một mình như một intransitive verb (như "hoạt động" được --> OK thay bằng "hoạt động".

- "car" mình sẽ update. Cái này không quan trọng, nếu đã có dịch trước rồi thì cứ theo thôi ;) 

- Đúng là "môi trường yên tĩnh" thì có vẻ quen tai hơn. Thực ra nếu chi li trong context thu âm thì có thể "background" có sắc thái riêng. Dù sao cũng không quan trọng nên mình sẽ đổi thành "môi trường"